lib/OpenQA/WebSockets/Controller/API.pm

```diff
@@ -17,16 +17,21 @@ package OpenQA::WebSockets::Controller::API;
 use Mojo::Base 'Mojolicious::Controller';
 
 use OpenQA::WebSockets;
+use OpenQA::WebSockets::Model::Status;
 
 sub is_worker_connected {
-    my $self      = shift;
-    my $worker_id = $self->param('worker_id');
-    my $bool      = OpenQA::WebSockets::ws_is_worker_connected($worker_id);
-    $self->render(json => {connected => $bool ? \1 : \0});
+    my ($self) = @_;
+
+    $self->render(
+        json => {
+            connected => OpenQA::WebSockets::Model::Status->singleton->is_worker_connected($self->param('worker_id'))
+            ? \1
+            : \0
+        });
```

Comment:
maybe someone created the local variables to prevent this ugly format ;) I prefer the former but ok